lib/hex/stdlib.ex

```diff
@@ -117,53 +117,57 @@ defmodule Hex.Stdlib do
   end
 
   # TODO: Remove this once we require OTP 22.1
-  if Code.ensure_loaded?(:crypto) and function_exported?(:crypto, :mac, 4) do
-    def crypto_hmac(type, key, data), do: :crypto.mac(:hmac, type, key, data)
-  else
-    def crypto_hmac(type, key, data), do: :crypto.hmac(type, key, data)
+  def crypto_hmac(type, key, data) do
+    if Code.ensure_loaded?(:crypto) and function_exported?(:crypto, :mac, 4) do
+      apply(:crypto, :mac, [:hmac, type, key, data])
```

Comment:
wdyt about using the new API without apply, i.e.:

```suggestion
      :crypto.mac(:hmac, type, key, data)
```

I believe this should compile on old OTPs, just emit a compile-time warning, right?

but totally fine as is!

Reply:
Yes, it will compile but may emit a warning on some versions. I want to reduce the amount of irrelevant warnings so the relevant ones don't get lost in the noise.

Reply:
good call!